<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ModAdminDialog</class>
 <widget class="QDialog" name="ModAdminDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>修改行政区信息</string>
  </property>
  <widget class="QPushButton" name="ptnCancel">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>310</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>50</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>区域编号*：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="edtAdminNo"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="label_6">
      <property name="styleSheet">
       <string notr="true">color:red</string>
      </property>
      <property name="text">
       <string>由0-9的数字组成，固定长6位</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QSplitter" name="splitter_2">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>110</y>
     <width>221</width>
     <height>161</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <widget class="QWidget" name="layoutWidget_5">
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>区域名称*：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="edtAdminName"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_6">
    <layout class="QHBoxLayout" name="horizontalLayout_7" stretch="1,3">
     <item>
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>区域类型：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cmbAdminType">
       <item>
        <property name="text">
         <string>省</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>市</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>区（县）</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_7">
    <layout class="QHBoxLayout" name="horizontalLayout_8">
     <item>
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>备     注：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="edtAdminComments"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QPushButton" name="ptnMod">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>310</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>修改</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
